Makefile

```diff
@@ -137,15 +137,11 @@ coverage: $(TEST_BUILD_DIR)/coverage.out
 	@echo "Checking code coverage"
 	@$(GORUN) $(GOTESTCOVERAGE) --config=./.testcoverage.yaml
 
-build-mock-management-plane-http:
-	mkdir -p $(BUILD_DIR)/mock-management-plane-http
-	@CGO_ENABLED=0 GOARCH=$(OSARCH) GOOS=linux $(GOBUILD) -o $(BUILD_DIR)/mock-management-plane-http/server test/mock/http/cmd/main.go
-
 build-mock-management-plane-grpc:
 	mkdir -p $(BUILD_DIR)/mock-management-plane-grpc
 	@CGO_ENABLED=0 GOARCH=$(OSARCH) GOOS=linux $(GOBUILD) -o $(BUILD_DIR)/mock-management-plane-grpc/server test/mock/grpc/cmd/main.go
 
-integration-test: $(SELECTED_PACKAGE) build-mock-management-plane-http build-mock-management-plane-grpc
+integration-test: $(SELECTED_PACKAGE) build-mock-management-plane-grpc
```

Comment:
I'll add a task for us for fixing our Makefile for some of these targets. Imo using phoney targets (e.g. `build-mock-management-plane-grpc` and `build-mock-management-plane-http`) as dependencies is not great practice if you can avoid it (which we can in this case).

Simple reference: https://stackoverflow.com/questions/2270643/what-is-a-make-target